gunicorn/config.py

```diff
@@ -1811,3 +1819,16 @@ class PasteGlobalConf(Setting):
 
         .. versionadded:: 20.0
         """
+
+class SpawnWorkerSleepTime(Setting):
+    name = "spawn_worker_sleep_time"
+    section = "Worker Processes"
+    cli = ['--spawn-worker-sleep-time']
```

Comment:
FWIW I prefer descriptive names over short names, so a slight +1 for `--spawn-worker-sleep-time`.  (`--spawn-worker-sleep` sounds like it could be a boolean flag, in any case.)